Secp256k1.Net/Secp256k1.cs

```diff
@@ -327,6 +329,30 @@ public bool SignatureNormalize(Span<byte> normalizedSignatureOutput, Span<byte>
             }
         }
 
+        /// <summary>
+        /// Parse a DER ECDSA signature
```

Comment:
Same comment about full description https://github.com/bitcoin-core/secp256k1/blob/e96901a4b951dfb6336114d223151dd0f5ffb89b/include/secp256k1.h#L359

Reply:
I have added the comment as requested